sherpa/astro/plot.py

```diff
@@ -476,6 +458,120 @@ def prepare(self, arf, data=None):
                 self.xhi = hc / self.xhi
 
 
+class RMFPlot(shplot.HistogramPlot):
+    """Create plots of the ancillary response file (RMF).
+
+    Attributes
+    ----------
+    xlo, xhi : array_like
+        The lower and upper edges of each bin.
+    y : array_like
+        The response for a specific energy or channel.
+    xlabel, ylabel, title : str
+        Plot labels.
+    labels : list of str
+        The labels for each line.
+    """
+
+    histo_prefs = shplot.basicbackend.get_model_histo_defaults()
+
+    # TODO: Make that a plot preference
+    # How many monochromatic lines to use
+    n_lines = 5
+
+    # Because this derived from NoNewAttributesAfterInit we need to
+    # make the attribute here it cna be used in prepare
```

Comment:
```suggestion
    # make the attribute here it can be used in prepare
```

My typing skillz strike again....